internal/code-generator/main.go

```diff
@@ -117,10 +122,12 @@ func main() {
 		{generator.Commit, "", "", commitFile, commitTmplPath, commitTmpl, commit},
 		{generator.MimeType, languagesYAML, "", mimeTypeFile, mimeTypeTmplPath, mimeTypeTmpl, commit},
 		{generator.Colors, languagesYAML, "", colorsFile, colorsTmplPath, colorsTmpl, commit},
+		{generator.Colors, languagesYAML, "", languagesFile, languagesTmplPath, languagesTmpl, commit},
```

Comment:
I belive re-using `generator.Colors` here would result in doing extra work of building a color-map, etc.

Shall we add a new, very simple generator function like `Languages` or `LanguageConstants`, implementing [generator.File](https://github.com/go-enry/go-enry/blob/master/internal/code-generator/generator/generator.go#L20) ?